app/validation/messages.py

```diff
@@ -0,0 +1,13 @@
+from flask.ext.babel import gettext as _
+
+# Set up default error and warning messages
```

Comment:
Why messages not errors? These are all error states plus soft checks are still errors (of sorts). When I saw this I instantly thought it was to do with translation.